lib/iex/lib/iex/evaluator.ex

```diff
@@ -186,6 +186,7 @@ defmodule IEx.Evaluator do
 
       # Evaluate the contents in the same environment server_loop will run in
       env = :elixir.env_for_eval(state.env, file: path, line: 1)
+      Process.put(:iex_imported_paths, MapSet.new([path]))
       {_result, binding, env, _scope} = :elixir.eval_forms(quoted, state.binding, env)
```

Comment:
And we clear it next, wdyt?

Reply:
We can probably clear it in an `after` clause after catch, which means we don't need to clean it when we raise in the helpers.

Reply:
good call, let me try that